crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -390,6 +390,18 @@ impl<'de> Deserialize<'de> for Entity {
     }
 }
 
+impl fmt::Debug for Entity {
```

Comment:
Nit: we could add a doc comment on this impl to specify what the `Debug` output signifies. It's not entirely clear unless you look into the source code

Reply:
Added documentation along with a sample serialized output to help users understand where to look for raw bits.